eiffel-syntax-and-usage/event-design-guidelines.md

```diff
@@ -34,3 +34,30 @@ The design of Eiffel events is governed by the following guidelines:
 * __Key names in camelCase:__ All key names shall use camelCase.
 * __Enumerations in CAPS_WITH_UNDERSCORE:__ All enumerated values shall use CAPS_WITH_UNDERSCORE.
 * __Link types are nouns:__ Link types shall have names that fit the pattern "\<target event> is the \<source event>'s \<link type>", implying that they are nouns that describe the relationship between the source and the target event.
+
+## Benefits and Implications
+
+By following the design guidelines the Eiffel protocol provides a language that
+gives producer and consumer a common way of understanding and describing the
+occurrences in the system by defining a contract between them.
+
+Following the guidlines stated, they adress the following issues:
+
+* __Dialects__: By using the protocol it gives the best possible conditions
+  for a non-initiated event consumer to make sense of any given set of events.
+* __Inconsistency__: By providing clear explanations of each parameter for both
+  producer and consumer it minimizes the risk for both data divergence and
+  inconsistency.
+* __Coordination__: The Eiffel protocol minimizes coordination effort between
+  producer and consumer by giving a well-defined specification by defining a
+  contract between the producer and the consumer.
```

Comment:
This sentence doesn't read too well (two subclauses beginning with "by XXXing"). Something like this?

> The Eiffel protocol minimizes coordination effort between producers and consumers by defining a specification that the parties can use as a contract.

Reply:
Thanks, reworded it based on you suggestion

Reply:
Better, but there are a couple of remaining nits. Suggestion:

The Eiffel protocol minimizes coordination efforts between  producers and consumers by providing a well-defined specification that the  parties can use as a contract.

Reply:
Fixed in [5de3f4b](https://github.com/eiffel-community/eiffel/pull/305/commits/5de3f4b54e8307affa3cf520065c8009c27434f7) ( and do hope I got it this time)